x-pack/legacy/plugins/spaces/server/routes/api/__fixtures__/create_test_handler.ts

```diff
@@ -8,6 +8,7 @@ import * as Rx from 'rxjs';
 import { Server } from 'hapi';
 import { Legacy } from 'kibana';
 import { KibanaConfig } from 'src/legacy/server/kbn_server';
+// eslint-disable-next-line @kbn/eslint/no-restricted-paths
```

Comment:
I was under the impression that this was the "correct" way to import mocks. Is there another approach we should take? Are mocks not meant to be used in this way at all?

Reply:
I guess it is still as "correct" as it was before. The difference is that before this PR `@kbn/eslint/no-restricted-paths` linter rule ignored Webpack aliases like `src/..`, now it processes them as well.

---

Maybe @elastic/kibana-platform could comment what is the "correct" way to import mocks/types? ^^

Reply:
yeah, unfortunately linter rule doesn't work with absolute paths yet. https://github.com/elastic/kibana/issues/36096
as a temporary solution you can change in .eslintrc
```diff
- '!src/core/server/mocks.ts',
+ '!src/core/server/mocks',

- '!src/core/public/mocks.ts',
+ '!src/core/public/mocks',
```